### PR TITLE
feat: use ArgparserAlias for xontrib

### PR DESCRIPTION
**Pull request by jnoortheen**
## For community
⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

### CODE REVIEW

File: xonsh/xontribs.py [hunk 1]
```diff
@@ -78,8 +79,38 @@ def update_context(name, ctx=None):
     return ctx.update(modctx)
 
 
-def xontribs_load(names, verbose=False):
-    """Load xontribs from a list of names"""
+def xontrib_names_completer(**_):
```
Comment (daniel-shimon) [inline xonsh/xontribs.py:82]: Can you add a test for this?

File: xonsh/xontribs.py [hunk 2]
```diff
@@ -118,11 +144,11 @@ def xontrib_installed(names: tp.Set[str]):
     return installed_xontribs
 
 
-def xontrib_data(ns):
+def xontrib_data(names=()):
     """Collects and returns the data about xontribs."""
     meta = get_xontribs()
     data = {}
-    names: tp.Set[str] = set() if not ns else set(ns.names)
+    names: tp.Set[str] = set(names or ())
```
[the remark this answers is not in the document]
Comment (jnoortheen) [reply xonsh/xontribs.py:151]: sometimes, the value from ns would be None. this is to avoid.

File: xonsh/xontribs.py (hunk 1 above)
Comment (jnoortheen) [reply xonsh/xontribs.py:82]: added

**Review by daniel-shimon — APPROVED** [review]
Cool!

**Pull request merged**